src/AutoMapper/Mappers/CollectionMapper.cs

```diff
@@ -44,35 +44,6 @@ public static class CollectionMapperExtensions
             return resolutionContext.Map(item, default(TDestinationItem));
         }
 
-
```

Comment:
Don't forget to remove MapItemMethodInfo, MapKeyValuePairMethodInfo, MapItemFunc, and MapKeyValuePairFunc since they tie in with MapCollection.